Jaehun

KwakOri opened the pull request.

KwakOri committed:
Type : Feat

Add a Card component

KwakOri committed:
Type : Add

Add dummy datas and modify Card component style

KwakOri committed:
Type : Feat

Add an authSlice

KwakOri committed:
Type : Feat

Set a protected route and github login

KwakOri committed:
Type : Feat

Add a side bar and a nav bar. modify login and authentication logic.

Minsu1322 commented:
최근커밋내에서 isLoading이  무슨기능인지 간단한 주석정도 달아주면 이해하기 빠를것같습니다!
아직 데이터베이스쪽을 안봐서 그부분은 잘 모르겠지만, 라우터부분 고생하셨습니당
남의 코드를  처음봐서 커밋내역을 왔다갔다 하다보니까 어렵네용 ㅎㅎ..


figma설계하면서 정했어야했는데  재훈님이 하신것처럼 styled-reset같은 css를 먼저 초기화하고 작성을 하면 좋을것같아요
추가로 App.jsx의 리액트쿼리부분과 vite.config.js의 @vitejs/plugin-react'; 부분에 경로단축설정?부분에서 오류가 나더라구용 이런부분도 맞추면 좋을것 같습니다. vite.config.js파일차이인것같기도 합니다!

Minsu1322 commented on src/pages/Layout/Layout.jsx:
```diff
@@ -1,10 +1,33 @@
 
 export const Layout = () => {
-  return (
+  const [isLoading, setLoading] = useState(true);
```
말씀해주신 부분입니다!

JaraNeck reviewed (approved):
좋습니다

JaraNeck reviewed (approved):
좋습니다

JaraNeck reviewed (approved):
좋습니다

JaraNeck reviewed (approved):
좋습니다

JaraNeck reviewed (approved):
좋아요

hwangshyun commented:
좋아용

KwakOri committed:
Type : Feat

로그아웃 버튼을 네비바로 옮겼습니다

KwakOri committed:
Type : Feat

Protected Route에 "/create-profile" 경로 추가
profile이 존재하지 않을 경우 "/create-profile"로 이동

KwakOri committed:
Type : None

KwakOri committed:
save

KwakOri committed:
save

KwakOri committed:
solve a conflict

KwakOri committed:
Merge branch 'dev' into jaehun

KwakOri committed:
add alias

KwakOri committed:
save
